README.md

```diff
@@ -42,22 +42,17 @@ These projects are located in the following repositories:
 * <https://github.com/nodejs/secrets> — _private_, administration delegated to the Build Working Group
 * <https://github.com/nodejs/Release> — administration delegated to the Release Working Group
 
-### Adjacent Projects
+### Adjacent projects
 
-The TSC is responsible for a number of projects that are not strictly required
-to plan, test, build, document and ship Node.js releases. Projects that are
-_adjacent_ are either created from within the technical organization managed by
+The TSC is responsible for several projects that are not strictly required
+to plan, test, build, document, and ship Node.js releases. Adjacent projects are either created from within the technical organization managed by
```

Comment:
nit: this could be split into two lines
```suggestion
to plan, test, build, document, and ship Node.js releases. Adjacent projects are
either created from within the technical organization managed by
```